scripts/eject.js

```diff
@@ -104,6 +104,11 @@ prompt('Are you sure you want to eject? This action is permanent. [y/N]', functi
   });
   delete hostPackage.scripts['eject'];
 
+  // explicitly specify ESLint config path for editor plugins
+  hostPackage.eslintConfig = {
+    extends: "./config/eslint.js",
```

Comment:
Yeah, `/` is fine in Node.js as an input path like this.

You only get problems if you're doing path stuff manually with `/` and you might have a path which has been made "native" by converting it to absolute or relative with `path` (which is why checking absolute paths with RegExps can be a gotcha - :musical_note: Gotta `[\\/]` 'Em All, Pathémon :musical_note:)